docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -54,8 +54,25 @@ proposal because most of the changes described below are a consequence of the wo
 
 ## API Changes
 
+- APIs have been moved to the top-level `api` folder (https://github.com/kubernetes-sigs/cluster-api/pull/12262).
```

Comment:
Adding the apiGroups in a follow-up PR: https://github.com/kubernetes-sigs/cluster-api/pull/12266